src/benchmarks/beluga_vs_nav2_multi_dataset/scripts/nominal.robot

```diff
@@ -0,0 +1,210 @@
+TorWIC SLAM               ${{TORWIC_SLAM_BAGS}}           ${{TORWIC_SLAM_DIR}}      ${{LASER_MODELS}}     odom           map        base_link       /front/scan    5              0.0             0.0             0.0                nav2_amcl::DifferentialMotionModel
+Willow Garage             ${{WILLOW_DSET_BAGS}}           ${{WILLOW_DSET_DIR}}      ${{LASER_MODELS}}     odom_combined  map        base_footprint  /base_scan     3              0.0             0.0             0.0                nav2_amcl::DifferentialMotionModel
+Diff Drive Sim 24hs       ${{DIFF_DRIVE_SIM_BAGS}}        ${{LONG_DURATION_DIR}}    ${{LASER_MODELS}}     odom           map        base_link       /scan          1              0.0             2.0             0.0                nav2_amcl::DifferentialMotionModel
+Omni Drive Sim 24hs       ${{OMNI_DRIVE_SIM_BAGS}}        ${{LONG_DURATION_DIR}}    ${{LASER_MODELS}}     odom           map        base_link       /scan          1              3.2             9.0             0.7                 nav2_amcl::OmniMotionModel
+
+*** Keywords ***
+Beluga vs Nav2 benchmark suite
+    Extends ROS 2 system benchmark suite
+    Extends ROS 2 2D SLAM system benchmark suite
+    Extends generic resource usage benchmark suite
+    Generates latexpdf report from nominal_report template in beluga_vs_nav2_multi_dataset ROS 2 package
+
+Beluga vs Nav2 benchmark case
+    Extends generic resource usage benchmark case
+    Extends ROS 2 system benchmark case
+    Extends ROS 2 2D SLAM system benchmark case
+    # Setup benchmark inputs
+    ${bagfile_path} =  Set Variable  ${EXECDIR}/${basedir}/bagfiles/${dataset}
+    ${artifacts_path} =  Set Variable  ${EXECDIR}/${basedir}/artifacts/${dataset}
+    Uses ${bagfile_path} at 1x as input to ROS 2 system
+    ${package_share_path} =  Find ROS 2 Package  beluga_vs_nav2_multi_dataset  share=yes
+    ${qos_override_path} =  Join Path  ${package_share_path}  config  qos_override.yml
+    Configures QoS overrides from ${qos_override_path} for input to ROS 2 system
+    # Setup benchmark rig
+    Uses beluga_vs_nav2_multi_dataset.launch in beluga_vs_nav2_multi_dataset ROS package as rig
+    Sets map_filename launch argument to ${artifacts_path}/map.yaml
+    Sets laser_model_type launch argument to ${laser_model}
+    Sets global_frame_id launch argument to ${map_frame}
+    Sets odom_frame_id launch argument to ${odom_frame}
+    Sets base_frame_id launch argument to ${base_frame}
+    Sets scan_topic launch argument to ${scan_topic}
+    Sets initial_pose_x launch argument to ${initial_pose_x}
+    Sets initial_pose_y launch argument to ${initial_pose_y}
+    Sets initial_pose_yaw launch argument to ${initial_pose_yaw}
+    Sets robot_model_type launch argument to ${robot_model_type}
```

Comment:
@glpuga meta: thinking out loud, LAMBKIN could use a mechanism for forward entire collections of arguments to an underlying launch file, automagically.

Reply:
As long as its just less verbose, but not less explicit, that's ok.

My personal point of view is that the more automagic things this does, the harder it is to make sense of how it works.

Reply:
>  the more automagic things this does, the harder it is to make sense of how it works.

That can be true, yes. In this case, perhaps a macro to invoke forwarding would at least require the user to explicitly request this. Logs may also help (and we are a bit loose in that sense in general, the troubleshooting experience is not great right now). Anyways, just a thought, nothing to do here.